independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/headers/NewCookieHeaderDelegate.java

```diff
@@ -59,6 +60,8 @@ public Object fromString(String newCookie) throws IllegalArgumentException {
                 version = Integer.parseInt(value);
             } else if (name.equalsIgnoreCase("HttpOnly")) {
                 httpOnly = true;
+            } else if (name.equalsIgnoreCase("SameSite")) {
+                sameSite = NewCookie.SameSite.valueOf(value.toUpperCase());
```

Comment:
What happens if a value different than the ones in the `SameSite` enum is passed along? The request fails or should we catch here and assume the value is `null`?

Reply:
It fails just like Max-age fails. I don't think we should catch the exception (and neither does Jersey)